spec/features/npq/npq_payment_calculation.feature

```diff
@@ -87,7 +89,7 @@ Feature: NPQ single-qualification payment schedule calculation
     Then expected variable payments should be as above
       And the service fee payment schedule should be:
         | Month | Service Fee |
-        | 1     | £25,612.90  |
+        | 1     | £25,613.00  |
```

Comment:
Should we have an example that rounds up incorrectly and requires a deduction from the first Service Fee?